packages/tasks/src/tasks/chat-completion/inference.ts

```diff
@@ -18,6 +18,10 @@ export interface ChatCompletionInput {
 	 * decreasing the model's likelihood to repeat the same line verbatim.
 	 */
 	frequency_penalty?: number;
+	/**
+	 * A guideline to be used in the chat_template
+	 */
+	guideline?: string;
```

Comment:
Not a fan of this argument as it's not used anymore. Best solution would be to remove it from TGI specs (see [slack thread](https://huggingface.slack.com/archives/C05CFK1HM0T/p1732009523291719?thread_ts=1724852501.099809&cid=C05CFK1HM0T)). Let's wait without merging this PR for now.

Reply:
opened https://github.com/huggingface/text-generation-inference/pull/2762 to remove `guideline` from TGI API